lib/src/Base/Type/openturns/Point.hxx

```diff
@@ -55,6 +55,9 @@ public:
   /** Constructor from a collection */
   Point(const Collection<Scalar> & coll);
 
+  /** Constructor from a collection */
+  explicit Point(const Collection<UnsignedInteger> & coll);
```

Comment:
I'm not sure about this one where do you need it ?

Reply:
In https://github.com/openturns/openturns/pull/1088/commits/3c1bafc2817ac1154019864e0339ed2dd833c0f2

Reply:
And a I agree I think it should be dropped

Reply:
Ok